packages/@react-spectrum/s2/src/TableView.tsx

```diff
@@ -784,20 +790,29 @@ function ResizableColumnContents(props: ResizableColumnContentProps) {
           </div>
           <Chevron size="M" className={chevronIcon} />
         </Button>
-        <Menu onAction={onMenuSelect} items={items} styles={style({minWidth: 128})}>
-          {(item) => <MenuItem>{item?.label}</MenuItem>}
+        <Menu onAction={onMenuSelect} styles={style({minWidth: 128})}>
+          {items.length > 0 && (
+            <MenuSection aria-label='Sort or resize?'>
```

Comment:
what's a label that actually makes sense here? multiple labels depending on if it's one or the other or both?

Reply:
Shouldn't this be a string key for i18n?
What about something nonspecific "Column options"? There will only be actions we take on a column, right?

Reply:
yes, i'm trying to figure out what it should read first though, or if we even want to have this be a section